rules/xcodeproj.bzl

```diff
@@ -334,12 +334,31 @@ def _xcodeproj_impl(ctx):
             "optional": True,
             "compilerFlags": "-fno-objc-arc",
         } for s in target_info.non_arc_srcs.to_list()]
-        asset_sources = [{
-            "path": paths.join(src_dot_dots, s.short_path),
-            "group": paths.dirname(s.short_path),
-            "optional": True,
-            "buildPhase": "none",
-        } for s in target_info.asset_srcs.to_list()]
+
+        asset_sources = []
+        for s in target_info.asset_srcs.to_list():
+            # https://github.com/bazelbuild/rules_apple/blob/b35ed275931a316b34089fc7fa5e1f20fbf8a15d/apple/internal/partials/support/resources_support.bzl#L157
+            # https://github.com/CocoaPods/CocoaPods/blob/43337555852797b2768ececba5e9ae0f8b896a58/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb#L245
+            if s.short_path.find(".xcdatamodeld/") != -1:
```

Comment:
do we similiarly need to handle `.xcassets/` or `.xcstickers/` directories ?

Reply:
yup they are being taken care in a separate PR